### DISCUSSION TITLE
update images within kustomize, CRDS, k8s manifests

### DISCUSSION THREAD

**Discussion by declanlong** [renovatebot/renovate#15879] [Request Help]
Hi,

I am using renovate to frequently rebuild images, which is working great, now I am trying to automate picking up these images within our kustomize, CRDS, helm values and k8s manifests.

Is this a use case which will is covered by renovate, any advice on how to approach this?

thanks in advance for your help :)

**Comment by rarkins** [comment]
We support at least 3 of those as managers (check the docs for those keywords). For any deeper answers, examples or reproduction repos would be best to demonstrate what you mean
